consensus/aura/aura.go

```diff
@@ -529,10 +510,12 @@ func (a *Aura) Prepare(chain consensus.ChainReader, header *types.Header) error
 	}
 	header.Extra = header.Extra[:extraVanity]
```

Comment:
This and line line 520 seem unnecessary now, we only store the signature in header.Extra, vanity does not exist